src/util/setSelectionsAndFocusEditor.ts

```diff
@@ -46,48 +55,47 @@ export async function focusEditor(editor: TextEditor) {
   } else {
     // If the view column is null we see if it's a notebook and try to see if we
     // can just move around in the notebook to focus the correct editor
-    const activeTextEditor = window.activeTextEditor;
 
-    if (activeTextEditor == null) {
-      return;
+    if (lt(version, "1.68.0")) {
```

Comment:
Might be overkill, but it might make sense to store this in a constants file. We should likely check `lt` and also ensure `gt` from some baseline.